lib/Notes/components/AssignmentsList/AssignmentsList.css

```diff
@@ -1,5 +1,5 @@
 @import "@folio/stripes-components/lib/variables.css";
 
 .assignments-list__item {
-  color: var(--primary);
+  color: var(--color-text);
```

Comment:
Can we just remove this file and thus the corresponding import instead? I have a feeling the default styling would be fine then and less likely to cause confusion down the road.

Reply:
Thanks for your feedback, updated.